p2p/peer_behaviour.go

```diff
@@ -0,0 +1,137 @@
+package p2p
+
+import (
+	"errors"
+	"sync"
+)
+
+// Type of error behaviour a peer can perform.
```

Comment:
```suggestion
// ErrorPeerBehaviour are types of observable erroneous peer behaviours.
```

This adheres to common go doc format.